src/main/java/it/gov/pagopa/pu/debtpositions/service/BaseDebtPositionOperationService.java

```diff
@@ -104,6 +110,13 @@ protected WorkflowCreatedDTO invokeWorkflow(DebtPositionDTO debtPositionDTO, Pay
     return null;
   }
 
+  /**
+   * Enable or disable the check of the flag on DebtPositionTypeOrg
+   *
+   * @return true if the check is enabled, false otherwise
+   */
+  protected abstract boolean isDebtPositionTypeOrgFlagCheckEnabled();
```

Comment:
I find this name ambigous, it seems that it should check if bypass entirely the check on DBTypeOrg (we are checking the relationship between the operator and the DBTypeOrg...
I would rename it in order to explicit the check on the DBTypeOrg disabled status